spring-boot/autoconfigure/src/main/java/com/linecorp/armeria/spring/ArmeriaAutoConfiguration.java

```diff
@@ -121,6 +128,12 @@ public Server armeriaServer(
 
             server.serviceAt(bean.getPath(), service);
             docServiceRequests.addAll(bean.getExampleRequests());
+            THttpService beanService = bean.getService().as(THttpService.class).get();
```

Comment:
Ah, it can change for now. Since armeria thrift service will focus on thrift-over-http, IMHO, we now don't need to care about non-THttpService.